core/src/tangram.cpp

```diff
@@ -229,7 +229,7 @@ void applySceneUpdates() {
 
```

Comment:
Here the shared_ptr<Scene> is copied so ```scene``` remains valid during ```applyConfig```. There is a minimal race-condition though as the copying of m_nextScene is not synchronized